src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String FIRST_SYMBOL = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        BufferedReader bufferedReader = null;
+        StringBuilder stringBuilder = null;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(fileName));
+            stringBuilder = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = bufferedReader.readLine();
+            }
```

Comment:
```suggestion
            String value;
            while (value  = bufferedReader.readLine() != null) {
                stringBuilder.append(value).append(System.lineSeparator());
            }
```

Reply:
Lead to compile error. Should i find solution with this or just ignore this? Compiler require change type of "value" to boolean.